sources/source-Send_to_email.module

```diff
@@ -46,6 +46,10 @@ class Send_to_Email extends superfecta_base {
 			'type' => 'text',
 			'default' => 'Unknown'
 		),
+		'Use_HTML' => array(
+			'description' => _('HTML enabled email'),
+			'type' => 'checkbox',
```

Comment:
Hi @bman87 please add default value "off" as well to ensure its not getting enabled by default for all.

Reply:
I made the requested change.  I am new to git for pull requests, please let me know if there is anything else I need to do? Thanks!